packages/ui/lib/components/Display/Address.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect } from 'react'
+import {
+  FiCopy as CopyIcon,
+  FiExternalLink as ExternalLinkIcon,
+} from 'react-icons/fi'
+import useEns from '../../../../../unlock-app/src/hooks/useEns'
```

Comment:
ha! the `ui` package should _not_ reference `unlock-app` to avoid circular deps.